go/cmd/dolt/commands/sql_test.go

```diff
@@ -272,42 +272,77 @@ func TestInsert(t *testing.T) {
 			query: "insert into dne (id) values (00000000-0000-0000-0000-000000000005)", expectedRes: 1,
 		},
 		{
-			name: "insert one row",
-			query: `insert into people (id, name, age, is_married) values 
+			name:  "insert one row",
```

Comment:
These command line tests are good to have, but they really shouldn't be where the main body of functional tests live -- these are really more like smoke tests to make sure that everything is wired together correctly. The exhaustive correctness tests should live in the package being exercised. Basically, copy the pattern I did with sqlselect_test.go, and apply it to sqlinsert_test.go.